samples/quickstart.js

```diff
@@ -142,7 +145,30 @@ function main(projectId, region, clusterName, jobFilePath) {
   }
 
   quickstart();
-  // [END dataproc_quickstart]
 }
 
-main(...process.argv.slice(2));
+const args = require('yargs').options({
+  project_id: {
+    demandOption: true,
+    describe: 'Project to use for creating resources.',
+    type: 'string',
+  },
+  region: {
+    demandOption: true,
+    describe: 'Region where the resources should live.',
```

Comment:
I agree that in a standalone application this is an improvement (I actually am one of the main folks who's worked on yargs throughout the years, and love it), but @fhinkel's sample guidelines call for samples to be imbedable without any dependencies; which I think is a reasonable tradeoff.

tldr; we probably shouldn't introduce yargs.

Reply:
Removed yargs dependency